packages/mui-codemod/src/v6.0.0/styled/test-cases/BasicStyled.expected.js

```diff
@@ -0,0 +1,104 @@
+const FormHelperTextRoot = styled('p')(({
+  theme
+}) => (({
+  color: (theme.vars || theme).palette.text.secondary,
+  ...theme.typography.caption,
+  textAlign: 'left',
+
+  [`&.${formHelperTextClasses.disabled}`]: {
+    color: (theme.vars || theme).palette.text.disabled,
+  },
+
+  variants: [{
+    props: {
+      size: 'small'
+    },
+
+    style: {
+      marginTop: 4,
+    }
+  }, {
+    props: {
+      variant: 'contained',
+      size: 'small'
+    },
+
+    style: {
+        marginTop: 6,
+      }
+  }, {
+    props: (
+      {
+        disabled,
+        ownerState
+      }
+    ) => ownerState.size === 'small' &&
+      ownerState.variant === 'contained' &&
+      disabled,
+
```

Comment:
```suggestion
```
nit: too many empty lines.

Reply:
This is done. Recast, a dependency of jscodeshift, handles the print functionality, so I have to remove empty lines manually.